php_phongo.h

```diff
@@ -130,7 +153,9 @@ const mongoc_read_prefs_t*    phongo_read_preference_from_zval(zval *zread_prefe
 const mongoc_write_concern_t* phongo_write_concern_from_zval  (zval *zwrite_concern TSRMLS_DC);
 const php_phongo_query_t*     phongo_query_from_zval          (zval *zquery TSRMLS_DC);
 
-void php_phongo_server_to_zval(zval *retval, const mongoc_server_description_t *sd);
+php_phongo_server_description_type_t php_phongo_server_description_type(mongoc_server_description_t *sd);
+
+void php_phongo_server_to_zval(zval *retval, mongoc_server_description_t *sd);
```

Comment:
You ended up removing the "const" in front of the argument ```*sd```. Was that on purpose?